src/applications/gi/tests/e2e/03-vet-tec.e2e.spec.js

```diff
@@ -32,7 +31,7 @@ module.exports = E2eHelpers.createE2eTest(client => {
   GiHelpers.expectLocation(client, `/program-search`);
   GiHelpers.selectSearchResult(
     client,
-    UtilHelpers.createId(
+    GiHelpers.createId(
```

Comment:
Once `import environment from 'platform/utilities/environment';` is removed from `src/applications/gi/utils/helpers.js` changes can be reverted